src/net.c

```diff
@@ -307,13 +307,19 @@ static unsigned int psk_server_callback(SSL *ssl, const char *identity, unsigned
 }
 #endif
 
+void ssl_keylog_callback_func(const SSL *ssl, const char *line)
+{
+	log__printf(NULL, MOSQ_LOG_INFO, "SSL Keylog: %s", line);
```

Comment:
Is there any value in adding the client address/port to the log line, to allow distinguishing between different clients? It makes sense to me, but I haven't used this sort of functionality before. If so then using `SSL_get_rbio()` and `BIO_get_fd()` to get the socket, and tweaking `net__socket_get_address()` to also return the port would allow you to change the message to `TLS Keylog for %s:%d %s`.